aep/0131.yaml

```
@@ -16,7 +18,7 @@ rules:
       function: falsy
 
   aep-131-operation-id:
-    description: Verifies that the operation ID is of the form getresource
+    description: The operation ID should be get{resource-singular}.
```

```suggestion
    description: The operation ID should be Get{resource-singular}.
```

Nit: I think it should be capitalized. It'll match the gRPC name, and based on what I saw in the openapi generation the result is the same.